opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc

```diff
@@ -0,0 +1,62 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[ga-user-config]]
+=== User Creation and Configuration
+
+Only a user with admin privileges can create users and assign security roles to them. 
+We recommend creating a new admin user instead of using the default. 
+
+Ideally, each user account corresponds to a person, to help track who performs tasks in your {opennms-product-name} system.
+Assigning different link:#ga-role-user-management-roles[security roles] to each user helps restrict what tasks the user can perform. 
+
+In addition to local users, you can configure external authentication services including link:https://wiki.opennms.org/wiki/Spring_Security_and_LDAP[LDAP / LDAPS], link:https://wiki.opennms.org/wiki/Spring_Security_and_Radius[RADIUS], and link:https://wiki.opennms.org/wiki/Single_Sign_On[SSO].
+Configuration specifics for these services are outside the scope of this documentation.
+
+WARNING: Do not delete the default _admin_ and _rtc_ users.
+         The _rtc_ user is used for the communication of the Real-Time Console on the start page to calculate the node and service availability.
+
+[[ga-user-create]]
+==== Creating a User
+
+. Log in as a user with administrative permissions.
+. Click the gear icon in the top right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Users*.
```

Comment:
just consistency thing, I guess we did in other places something like this here:

```
"Configure OpenNMS -> Configure Users, Groups and On-Call roles"
```
This is the spot I've looked it up: https://github.com/OpenNMS/opennms/edit/develop/opennms-doc/guide-install/src/asciidoc/text/basic-setup-opennms/rhel/step-4-change-admin-password.adoc

Reply:
I was considering changing this to the > only as I updated the docs, but am fine with the ->